[Bug]: Cannot Load the Console UI

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Operating System

Ubuntu 22.04

### CKAN Version

1.33.2

### Game Version

irrelevant

### Did you make any manual changes to your game folder (i.e., not via CKAN)?

no

### Describe the bug

CKAN console UI failed to start, instead raised a .NET exception.

### Steps to reproduce

Type `ckan consoleui` on the command line.

### Relevant log output

```
System.TypeLoadException: Could not load type of field 'CKAN.ConsoleUI.Toolkit.ConsoleTextBox:lines' (5) due to: Could not load file or assembly 'System.ServiceModel, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' or one of its dependencies.
  at CKAN.ConsoleUI.GameInstanceListScreen.TryGetInstance (CKAN.ConsoleUI.Toolkit.ConsoleTheme theme, CKAN.GameInstance ksp, System.Action`1[T] render) [0x00038] in <1415add505c24fcdaa2e24ead75d4099>:0 
  at CKAN.ConsoleUI.SplashScreen.Run (CKAN.ConsoleUI.Toolkit.ConsoleTheme theme) [0x0001e] in <1415add505c24fcdaa2e24ead75d4099>:0 
  at CKAN.ConsoleUI.ConsoleCKAN..ctor (CKAN.GameInstanceManager mgr, System.String themeName, System.Boolean debug) [0x00035] in <1415add505c24fcdaa2e24ead75d4099>:0 
  at CKAN.ConsoleUI.ConsoleUI.Main_ (System.String[] args, CKAN.GameInstanceManager manager, System.String themeName, System.Boolean debug) [0x00005] in <1415add505c24fcdaa2e24ead75d4099>:0 
  at CKAN.CmdLine.MainClass.ConsoleUi (CKAN.GameInstanceManager manager, CKAN.CmdLine.ConsoleUIOptions opts, System.String[] args) [0x00034] in <1415add505c24fcdaa2e24ead75d4099>:0 
  at CKAN.CmdLine.MainClass.RunSimpleAction (CKAN.CmdLine.Options cmdline, CKAN.CmdLine.CommonOptions options, System.String[] args, CKAN.IUser user, CKAN.GameInstanceManager manager) [0x002df] in <1415add505c24fcdaa2e24ead75d4099>:0 
  at CKAN.CmdLine.MainClass.Execute (CKAN.GameInstanceManager manager, CKAN.CmdLine.CommonOptions opts, System.String[] args) [0x00296] in <1415add505c24fcdaa2e24ead75d4099>:0 
  at CKAN.CmdLine.MainClass.Main (System.String[] args) [0x000e9] in <1415add505c24fcdaa2e24ead75d4099>:0 
[ERROR] FATAL UNHANDLED EXCEPTION: System.TypeLoadException: Could not load type of field 'CKAN.ConsoleUI.Toolkit.ConsoleTextBox:lines' (5) due to: Could not load file or assembly 'System.ServiceModel, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' or one of its dependencies.
  at CKAN.ConsoleUI.GameInstanceListScreen.TryGetInstance (CKAN.ConsoleUI.Toolkit.ConsoleTheme theme, CKAN.GameInstance ksp, System.Action`1[T] render) [0x00038] in <1415add505c24fcdaa2e24ead75d4099>:0 
  at CKAN.ConsoleUI.SplashScreen.Run (CKAN.ConsoleUI.Toolkit.ConsoleTheme theme) [0x0001e] in <1415add505c24fcdaa2e24ead75d4099>:0 
  at CKAN.ConsoleUI.ConsoleCKAN..ctor (CKAN.GameInstanceManager mgr, System.String themeName, System.Boolean debug) [0x00035] in <1415add505c24fcdaa2e24ead75d4099>:0 
  at CKAN.ConsoleUI.ConsoleUI.Main_ (System.String[] args, CKAN.GameInstanceManager manager, System.String themeName, System.Boolean debug) [0x00005] in <1415add505c24fcdaa2e24ead75d4099>:0 
  at CKAN.CmdLine.MainClass.ConsoleUi (CKAN.GameInstanceManager manager, CKAN.CmdLine.ConsoleUIOptions opts, System.String[] args) [0x00034] in <1415add505c24fcdaa2e24ead75d4099>:0 
  at CKAN.CmdLine.MainClass.RunSimpleAction (CKAN.CmdLine.Options cmdline, CKAN.CmdLine.CommonOptions options, System.String[] args, CKAN.IUser user, CKAN.GameInstanceManager manager) [0x002df] in <1415add505c24fcdaa2e24ead75d4099>:0 
  at CKAN.CmdLine.MainClass.Execute (CKAN.GameInstanceManager manager, CKAN.CmdLine.CommonOptions opts, System.String[] args) [0x00296] in <1415add505c24fcdaa2e24ead75d4099>:0 
  at CKAN.CmdLine.MainClass.Main (System.String[] args) [0x000e9] in <1415add505c24fcdaa2e24ead75d4099>:0
```

--- a/debian/control.in
+++ b/debian/control.in
@@ -3,7 +3,7 @@ Version: @VERSION@
 Architecture: all
 Section: utils
 Priority: optional
-Depends: mono-runtime (>=5.0), ca-certificates-mono, libmono-microsoft-csharp4.0-cil, liblog4net1.2-cil, libnewtonsoft-json5.0-cil, libmono-system-net-http-webrequest4.0-cil
+Depends: mono-runtime (>=5.0), ca-certificates-mono, libmono-microsoft-csharp4.0-cil, liblog4net1.2-cil, libnewtonsoft-json5.0-cil, libmono-system-net-http-webrequest4.0-cil, libmono-system-servicemodel4.0a-cil
 Maintainer: The CKAN authors <debian@ksp-ckan.space>
 Description: KSP-CKAN official client.
  Official client for the Comprehensive Kerbal Archive Network (CKAN).

